jenkins-job-generator/definitions/sapmachine-build-jobs.yml

```diff
@@ -236,10 +236,23 @@
                                 }}
                             }}
                         }}
+            sign_and_notarize: &sign_and_notarize |-
+                stage("Sign and notarize artefacts") {{
+                            when {{
+                                expression {{ params.PUBLISH == true && JOB_NAME ==~ /((\S*)(-macos)(\S*)(-release))/ }}
+                            }}
+                            steps {{
+                                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'SapMachine-github', usernameVariable: 'GIT_USER', passwordVariable: 'GIT_PASSWORD'],
```

Comment:
I believe the github credential is not needed here.

Reply:
done